app/src/components/unbonding/UnbondingLog.tsx

```diff
@@ -21,43 +23,49 @@ export const UnbondingLog = ({ unbondingAccount }: { unbondingAccount: Unbonding
     return setIsUnbonded(unbondedState);
   }, [setIsUnbonded, unbondingAccount]);
 
+  const currentTime = useCurrentTime();
+  const oneDayCountdown =
+    !isUnbonded &&
+    bnToNumber(unbondingAccount.unbondingAccount.unbondedAt) * 1000 - currentTime < ONE_DAY;
+
+  const getUnbondingAccountExplorerUrl = () => {
+    unbondingAccount &&
+      window.open(
+        getAccountExplorerUrl(unbondingAccount.address.toBase58()),
+        "_blank",
+        "noreferrer"
+      );
+  };
+
   return (
     <tr>
-      <td
-        className="italics"
-        onClick={() =>
-          unbondingAccount &&
-          window.open(
-            getAccountExplorerUrl(unbondingAccount.address.toBase58()),
-            "_blank",
-            "noreferrer"
-          )
-        }
-      >
+      <td className="italics" onClick={getUnbondingAccountExplorerUrl}>
         {dateFromUnixTimestamp(unbondingAccount?.unbondingAccount.unbondedAt)}
       </td>
-      <td
-        className="italics"
-        onClick={() =>
-          unbondingAccount &&
-          window.open(
-            getAccountExplorerUrl(unbondingAccount.address.toBase58()),
-            "_blank",
-            "noreferrer"
-          )
-        }
-      >
-        Unbonding{" "}
-        <Tooltip
-          title="Unstaking transactions require a 29.5-day unbonding period. before withdrawal to your wallet is enabled. Status will show as 'unbonding' until this period completes."
-          mouseEnterDelay={0.1}
-        >
-          <InfoCircleFilled />
-        </Tooltip>
+      <td className="italics tooltip" onClick={getUnbondingAccountExplorerUrl}>
+        {/* TODO toggle */}
+        {!isUnbonded ? "Unbonded" : "Unbonding"}
+        {isUnbonded && (
+          <Tooltip
+            title="Unstaking transactions require a 29.5-day unbonding period. before withdrawal to your wallet is enabled. Status will show as 'unbonding' until this period completes."
+            mouseEnterDelay={0.1}
+          >
+            <InfoCircleFilled />
+          </Tooltip>
+        )}
       </td>
+
       <td className="action italics">
-        <i className="italics">
-          Unstake complete on {dateFromUnixTimestamp(unbondingAccount.unbondingAccount.unbondedAt)}
+        <i onClick={getUnbondingAccountExplorerUrl}>
+          {/* TODO toggle */}
+          {!oneDayCountdown
+            ? `Unstake complete in ${getRemainingTime(
+                currentTime,
+                // bnToNumber(unbondingAccount.unbondingAccount.unbondedAt) * 1000
+                1648859588 * 1000
```

Comment:
I see this was probably for testing, but let's not check it in
```suggestion
                bnToNumber(unbondingAccount.unbondingAccount.unbondedAt) * 1000
```